runtime/ops/os.rs

```diff
@@ -161,7 +161,7 @@ fn op_exit(state: &mut OpState) {
 #[op]
 fn op_loadavg(state: &mut OpState) -> Result<(f64, f64, f64), AnyError> {
   super::check_unstable(state, "Deno.loadavg");
-  state.borrow_mut::<Permissions>().env.check_all()?;
+  state.borrow_mut::<Permissions>().sys.check("loadavg")?;
```

Comment:
Could you update these checks to work similarly to https://github.com/denoland/deno/commit/212b7dd6da487c070229b6348ec7907b4fecbcf9 ?